src/Item.tsx

```diff
@@ -11,6 +11,9 @@ export const Item: FunctionComponent = (props) => {
   const request = useRequest(Number(itemNum));
   let requestRoles: RoleType[];
 
+  // Initialize it as not being an open request so that the Checklist Complete buttons do not render until we know if it is or not
+  let isRequestOpen: boolean = false;
+
```

Comment:
I moved this logic into the RequestApi itself, making a new internal field called "status".  I think this makes the most sense, since we will need to use this information with other components (such as the "My Checklist Items" and possibly the "My Requests").  I also went ahead and wrapped the Checklist component inside waiting for the data for the request since we now pass the request down to the checklist, and it doesn't really make much sense to render the checklist items as "Read Only" and then re-render them once we have the Status, since the user should have some reference info for what the checklist item is for.